src/main/resources/com/linecorp/armeria/server/docs/assets/js/armeria.js

```diff
@@ -24,6 +24,85 @@ $(function () {
     render(URI(window.location.href));
   });
 
+  function parseParametersDocString(docString) {
+    var parameters = {};
+    if (docString != null) {
+      var pattern = /@param ([^ \.]*)[ \.]*(.*)/g;
```

Comment:
Just in case the parameter description does not end with a dot (.), could we also stop at `@`? (but not at `{@`)